front/cypress/e2e/profile_edition.cy.ts

```diff
@@ -22,19 +22,10 @@ describe('profile edition', () => {
     cy.get('input[type="file"]').attachFile('icon.png');
     cy.get('#first_name').clear();
     cy.get('#last_name').clear().type('Doe');
-    const newEmail = randomEmail();
-    email = newEmail;
-    cy.get('#email').clear().type(newEmail);
     cy.get('button[type="submit"]').contains('Save changes').click();
     cy.wait('@saveUser');
     cy.get('[data-testid="feedbackSuccessMessage"]').should('exist');
   });
-  it('shows errors when fields have invalid values', () => {
-    cy.get('#email').clear();
-    cy.get('button[type="submit"]').contains('Save changes').click();
```

Comment:
We've moved Email out of this form, so this test isn't necessary anymore. We still need to add a new test for the new email update at some point though..